src/slack/events/appMention.ts

```diff
@@ -0,0 +1,19 @@
+import { App } from '@slack/bolt';
+import { Middleware, SlackEventMiddlewareArgs } from '@slack/bolt';
+import { app } from '..';
+import logger from '../../logger';
+
+export const appMention: Middleware<SlackEventMiddlewareArgs<'app_mention'>> = async ({ event, context, payload}) =>{
+
+  try {
+  app.client.chat.postMessage({
```

Comment:
Wrap this method in a try/catch:
```typescript
try {
  // The code above
} catch (err) {
  logger.error('Something went wrong trying to reply to a public mention: ', err);
}

Reply:
postMessage({}) is already in a try/catch, do you want us add a try/catch to the overall appMention as well?

Reply:
Your changes are sufficient, this looks good